### PR TITLE
Deprecate NormalizePermute in favor of CropMirrorNormalize

### PR DESCRIPTION
**Pull request by Kh4L**
Signed-off-by: Serge Panev <spanev@nvidia.com>

### CODE REVIEW

**Comment by Kh4L** [comment]
!build

File: dali/pipeline/operators/fused/normalize_permute.cc [hunk 1]
```diff
@@ -70,6 +70,9 @@ Normalization takes input image and produces output using formula
 ..
 
 output = (input - mean) / std
+
+*Deprecated*: Use CropMirrorNormalize_ (without crop and mirror arguments) instead,
```
Comment (JanuszL) [inline dali/pipeline/operators/fused/normalize_permute.cc]: Maybe we should crash inside the `NormalizePermute` constructor with the `Deprecated` message so use will know that he should move to `CropMirrorNormalize`.

Comment (JanuszL) [reply dali/pipeline/operators/fused/normalize_permute.cc]:
```suggestion
*Deprecated*: Use :meth:`nvidia.dali.ops.CropMirrorNormalize` (without crop and mirror arguments) instead,
```

Comment (Kh4L) [reply dali/pipeline/operators/fused/normalize_permute.cc]: Maybe only having a warning for this version, and make it crash on the next one?

Comment (JanuszL) [reply dali/pipeline/operators/fused/normalize_permute.cc]: Ok. Let's go with warning first.

Comment (Kh4L) [reply dali/pipeline/operators/fused/normalize_permute.cc]: Done

**Comment by Kh4L** [comment]
!build

File: dali/pipeline/operators/fused/normalize_permute.h [hunk 2]
```diff
@@ -31,6 +31,12 @@ class NormalizePermute : public Operator<Backend> {
     H_(spec.GetArgument<int>("height")),
     W_(spec.GetArgument<int>("width")),
     C_(IsColor(spec.GetArgument<DALIImageType>("image_type")) ? 3 : 1) {
+
+    std::cerr << "Warning: Deprecated: NormalizePermute: Use CropMirrorNormalize (without crop and "
```
Comment (jantonguirao) [inline dali/pipeline/operators/fused/normalize_permute.h]: DALI_WARN

**Comment by Kh4L** [comment]
!build

**Comment by Kh4L** [comment]
!build

File: dali/pipeline/operators/fused/normalize_permute.h [hunk 3]
```diff
@@ -32,6 +32,11 @@ class NormalizePermute : public Operator<Backend> {
     H_(spec.GetArgument<int>("height")),
     W_(spec.GetArgument<int>("width")),
     C_(IsColor(spec.GetArgument<DALIImageType>("image_type")) ? 3 : 1) {
+    DALI_WARN("Warning: Deprecated: NormalizePermute: Use CropMirrorNormalize (without crop and "
```
Comment (mzient) [inline dali/pipeline/operators/fused/normalize_permute.h]: Too many colons here...
```suggestion
    DALI_WARN("Warning: NormalizePermute is deprecatred. Use CropMirrorNormalize (without crop and "
```

**Comment by Kh4L** [comment]
!build

**Pull request merged**